packages/auto-label/src/auto-label.ts

```diff
@@ -0,0 +1,256 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import { Application, Octokit } from 'probot';
+import { GitHubAPI } from 'probot/lib/github';
+import * as path from 'path';
+import e = require('express');
+import { IssuesListLabelsOnIssueResponseItem } from '@octokit/rest';
+
+const fs = require('fs');
+const colorsData = JSON.parse(
+  fs.readFileSync(path.resolve(__dirname, '../../src/colors.json'), 'utf8')
+);
+
+interface JSONData {
+  num_total_prs: number;
+  num_open_p1s: number;
+  num_open_questions: number;
+  num_open_p0s: number;
+  language: string;
```

Comment:
In the future, you could be sneaky and only define the fields you care about in the interface.